### DISCUSSION TITLE
🥳 Script: Internet speed test

### DISCUSSION THREAD

**Discussion by dipakcg** [johnlindquist/kit#72] [Share]
Hello everyone,

Excited to share that I've released a script that checks internet upload/download speed. More about the script can be found here: https://github.com/dipakcg/script-kit-speedtest

_* I am working on WordPress – PHP for over a decade, but new to node and never written any such script. This is my first attempt at writing such script/code._ 

Thanks @johnlindquist for Script Kit.

**Comment by johnlindquist** [comment]
This is so great, thanks so much for sharing!

This also confirms to me that I need to put more work into that `show` dialog so you can push text updates :)
